cmd/triggerRun/cmd/root.go

```diff
@@ -131,15 +141,27 @@ func readTrigger(path string) ([]*v1alpha1.Trigger, error) {
 	return list, nil
 }
 
-func readHTTP(path string) (*http.Request, error) {
+func readHTTP(path string) (*http.Request, []byte, error) {
 	f, err := os.Open(path)
+	if err != nil {
+		return nil, nil, fmt.Errorf("error opening file: %w", err)
+	}
 
+	request, err := http.ReadRequest(bufio.NewReader(f))
 	if err != nil {
-		return nil, fmt.Errorf("error opening file: %w", err)
+		return nil, nil, fmt.Errorf("error reading HTTP file: %w", err)
 	}
+
+	body, err := ioutil.ReadAll(request.Body)
```

Comment:
I realize this is similar to what is already being done, but won't doing this consume the body in the request? Should we be using `request.GetBody()` instead?